### PR TITLE
Changed GeoPage datasource to db

### PR DESCRIPTION
**Pull request by Mnosek**

### CODE REVIEW

File: src/KitchenSink/GeoPage.json.cs [hunk 1]
```diff
@@ -1,56 +1,51 @@
...
+                };
+                Transaction.Commit();
+            }
+        }
+    }
+
+    [GeoPage_json.Position]
+    partial class GeoPagePosition : Json, IBound<SharedPosition> {
+        public void Handle(Input.Reset action) {
+            Latitude  = ((GeoPage)Parent).DefaultLatitude;
```
Comment (aloscha) [inline src/KitchenSink/GeoPage.json.cs]: Move parent with cast into another variable.

File: src/KitchenSink/GeoPage.json.cs [hunk 2]
```diff
@@ -1,56 +1,51 @@
...
+        protected void PushChanges() {
+            Session.ForAll((s, sessionId) => {
+                var master = s.Data as MasterPage;
+                var navpage = master?.CurrentPage as NavPage;
+                if (!(navpage?.CurrentPage is GeoPage)) return;
+                if ((GeoPage)navpage.CurrentPage != null) {
+                    s.CalculatePatchAndPushOnWebSocket();
+                }
+            });
+            Transaction.Commit();
```
Comment (aloscha) [inline src/KitchenSink/GeoPage.json.cs]: Maybe first commit transaction and then push to other

**Comment by Mnosek** [comment]
@aloscha please have a look

File: src/KitchenSink/wwwroot/KitchenSink/GeoPage.html [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
+            <google-map-marker drag-events="true" on-google-map-marker-dragend="onGoogleMapMarkerDrag" id="kitchensink-geopage-marker" draggable="true" title="Hi Stockholm!" latitude="{{model.Position.Latitude$}}" longitude="{{model.Position.Longitude$}}"></google-map-marker>
+        </google-map>
+        <button value="{{model.Position.Reset$::click}}" onclick="this.value++" class="btn btn-primary">Reset</button>
+    </template>
+    <script>
+        (function init(template) {
+            template.onGoogleMapMarkerDrag = function () {
+                template.set("model.Position.EventPositionChange$", template.model.Position.EventPositionChange$ + 1);
+            };
+        })(document.currentScript.previousElementSibling);
```
Comment (aloscha) [inline src/KitchenSink/wwwroot/KitchenSink/GeoPage.html]: Is it necessary `(document.currentScript.previousElementSibling)`? I think `currentScript` is `undefined`

Comment (Mnosek) [inline src/KitchenSink/wwwroot/KitchenSink/GeoPage.html]: I've changed whole js. Should be better now.

**Comment by aloscha** [comment]
For me it's okay, @warpech I think you could also see the result

**Comment by warpech** [comment]
As the result of my review, I have posted three changes (see above).

@Mnosek please review my changes. I can further explain if needed.

Merging to `master`.

**Pull request merged**